src/components/status-image/status-image.css

```
@@ -1,4 +1,21 @@
 .StatusImage {
+    
     border: 1px solid white;
-    background-color: var(--color-busy);
+    height: 423px;
+    width: 414px;
+}
+
+img {
+    position: absolute;
+    max-height: 423px;
+    max-width: 414px;
+    left: 0;
+}
+
+#person {
+    position: relative;
+    top: 0;
+    left: 0;
+    max-height: 368px;
+    max-width: 292px;
```

The parent has been changed to position relative. Now you can use position absolute. Place at the bottom and center horizontally. max-width/height should be 100% and the absolute value is for height and width.
```suggestion
    position: absolute;
    bottom: 0;
    left: 50%;
    transform: translateX(-50%);
    height: 368px;
    width: 292px;
    max-height: 100%;
    max-width: 100%;
```